packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -18,6 +18,7 @@ export const init = (_context: Context): void => {
  * Handle exceptions
  */
 export const handleException = async (exception: unknown): Promise<void> => {
+  process.exitCode = 1;
```

Comment:
why is this needed when we have a call at the end of this function ?

Reply:
I observed a case where if the exception handler errored out before it got to the `process.exit(1)` it would exit with 0 even though there was a failure. This ensures that if we enter the exception handler, we will exit with 1 regardless of what happens